pages/platform/ai/deploy_tuto_10_locust/guide.en-gb.md

```diff
@@ -0,0 +1,252 @@
+
+### Install locust.io
+
+Locust is an open source Python package, that you can install with one line of code. Follow their [official documentation](https://docs.locust.io/en/stable/index.html):
+
+```console
+pip3 install locust
+```
+
+You can install in on your personal computer, but keep in mind that load testing tools will require four things to **not become the bottleneck in your load test**:
+- Enough compute (CPU).
+- Enough memory (RAM).
+- Low latency connectivity to your API.
+- no "noisy neighbors", meaning no software installed that can compromises your results. imagine your CPU power getting used by video rendering, it will biases your results.
+
+For all these reasons, a **Cloud instance is recommended**, such as a medium-sized virtual machine. For this tutorial, we will take and OVHcloud instance B2-30.
+
+
+### Configure Locust
+
+To configure the software, you need to create a file named `locustfile.py`. In this file, you can put the path where you want to make your request, the headers of your request, the type of the request (POST, GET, etc) and the body if you want to add a body to the request. 
+
+A generic file will look like this:
+
+```python
+from locust import HttpUser, task
+
+class HelloWorldUser(HttpUser):
+    @task
+    def hello_world(self):
+        self.client.get("/hello")
+        self.client.get("/world")
+```
+
+For my API and my needs, the locust file will be slightly modified: 
+
+```python
+# Import the Locust dependencies
+from locust import HttpUser, task
+
+# Import general library from python
+import os
+import random
+
+# Import lorem ipsum library to generate some random texts
+from lorem_text import lorem
+
+# Import dotenv to load the environments variables
+from dotenv import load_dotenv
+load_dotenv()
+
+# Create a table with some lorem ipsum texts
+messages = []
+
+# Add 1000 random texts of 10 paragraphs each (simulate 1000 emails)
+for i in range(1000):
+    messages.append(lorem.paragraphs(10))
+
+# Define the headers of the request. Token is stored as an environment variable here
+headers = {
+    "Authorization": "Bearer {os.getenv('TOKEN')}", "Content-Type": "application/json"}
+
+
+class HelloWorldUser(HttpUser):
+    # Definition of the first path where we do our post request
+    @task
+    def hello_world(self):
+        # Define the body with the email choose randomly from the tab of all the emails
+        body = {"message": random.choice(messages)}
+        # Do the post request on the spam detection path
+        self.client.post("/spam_detection_path",
+                         headers=headers, json=body)
+```
+
+For your own needs, you will have to change the path, the headers and the body because these are parameters who change from one API to other API. 
```

Comment:
```suggestion
For your own needs, you will have to change the path, the headers and the body because these are parameters who change from one API to another. 
```